docs/community_resources/phpstorm_plugin.md

```diff
@@ -0,0 +1,173 @@
+# Ibexa DXP plugin for PhpStorm
+
+Ibexa DXP plugin for PhpStorm helps you to work with Ibexa DXP by speeding up installation
```

Comment:
```suggestion
Ibexa DXP plugin for PhpStorm helps you work with Ibexa DXP by speeding up installation
```

Reply:
Both phrases are correct and i don't believe there is a difference in meaning, and the version with "to" is easier to parse IMHO.